web/src/modules/common/components/Tooltip.js

```diff
@@ -0,0 +1,180 @@
+// @flow
```

Comment:
Why did decide to add is at standalone instead of using yarn package?

Reply:
This is my implementation :)